api/v1alpha1/volumesnapshotrestore_types.go

```diff
@@ -45,19 +47,31 @@ type VolumeSnapshotRestoreStatus struct {
 	StartTimestamp *metav1.Time `json:"startTimestamp,omitempty"`
 	// CompletionTimestamp records the time a volumesnapshotrestore reached a terminal state.
 	CompletionTimestamp *metav1.Time `json:"completionTimestamp,omitempty"`
-	// Includes information pertaining to Volsync ReplicationDestination CR
+	// Includes information pertaining to VolSync ReplicationDestination CR
 	ReplicationDestinationData ReplicationDestinationData `json:"replicationDestinationData,omitempty"`
 }
 
 type VSBRef struct {
 	// Includes backed up PVC name and size
-	BackedUpPVCData PVCData `json:"sourcePVCData,omitempty"`
+	BackedUpPVCData PVCData `json:"sourcePVCData,omitempty"` // this is from here api/v1alpha1/volumesnapshotbackup_types.go line 72, how it got imported?
```

Comment:
In Go you don't need to import things if they are in the same dir? (but their name need to start with uppercase, right?)